code/modules/clothing/masks/gasmask.dm

```diff
@@ -199,6 +199,48 @@
 		playsound(src.loc, 'sound/creatures/hoot.ogg', 50, 1)
 		cooldown = world.time
 
+/obj/item/clothing/mask/gas/tiki_mask
+	name = "tiki mask"
+	desc = "A creepy wooden mask. Surprisingly expressive for a poorly carved bit of wood."
+	icon_state = "tiki_eyebrow"
+	resistance_flags = FLAMMABLE
+	max_integrity = 100
+	actions_types = list(/datum/action/item_action/adjust)
+	dog_fashion = null
+	species_exception = list(/datum/species/golem/wood)
+	sprite_sheets = list()
+	var/list/tikimask_designs = list()
+
+/obj/item/clothing/mask/gas/tiki_mask/Initialize(mapload)
+	.=..()
+	tikimask_designs = list(
+		"Original Tiki" = image(icon = src.icon, icon_state = "tiki_eyebrow"),
```

Comment:
in this case, we really do have to use `src`, as gross as that is---and since this is all built in BYOND stuff, we can't really do the whole `icon = _icon` thing either).